Plots/MapProjections/NCL_sat_2.py

```diff
@@ -0,0 +1,371 @@
+"""
+NCL_sat_2.py
+===============
+This script illustrates the following concepts:
+   - Unpacking 'short' data
```

Comment:
Is this comment relevant for the Python version of this plot?

Reply:
I do change the float data into 'short' data, so I just changed the comment to "Converting float data into short data".